src/lib/pbo/quadratization.jl

```diff
@@ -23,7 +23,11 @@ aux(::Integer)::Vector{S}
 @doc raw"""
     Quadratization{NTR_KZFD}(stable::Bool = false)
 
-NTR-KZFD (Kolmogorov & Zabih, 2004; Freedman & Drineas, 2005)
+Negative Term Reduction NTR-KZFD (Kolmogorov & Zabih, 2004; Freedman & Drineas, 2005)
+
+```math
+-x_1 x_2 \dots x_k \to (k-1) x_{aux} - \sum_i x_i x_{aux}
```

Comment:
Isn't this missing the quadratization symbol for them? Something like:
\mathcal{Q}:(x_1, \dots, x_k) \to ...